test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -145,20 +238,23 @@ upgrade_downgrade()
         echo "Upgrade/downgrade testing with \"$1\" and \"$2\""
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 
+        file_name_1=$(get_config_file_name $1)
+        file_name_2=$(get_config_file_name $2)
```

Comment:
Can we use some more descriptive variable name e.g. `cfg_file_branch1`?

Reply:
And I think extracting the $1 and $2 into local variables with meaning names is a good practice (just as what you did for another function).

Reply:
Fixed